### DISCUSSION TITLE
Is it possible to run deno in jupyter notebook?

### DISCUSSION THREAD

**Discussion by y9c** [denoland/deno#2168] [Ideas]
I found Itypescript kernel, https://github.com/nearbydelta/itypescript.
But it is based on node.

**Comment by ry** [comment]
No it's not. Would be nice but it's probably not going to happen any time soon.

**Comment by josephrocca** [comment]
Could this be re-opened, and perhaps retitled so it's a feature request? Maybe better than creating a new issue since this one is already at the top of a Google search.

**Comment by kitsonk** [comment]
I don't think re-opening it would be useful, it would give someone false hope that it would ever be worked on.

**Comment by josephrocca** [comment]
Ah, okay. Is it the Deno team's position that a notebook environment wouldn't be useful/practical, or is it just something for a different (possibly unofficial) repo?

**Comment by kitsonk** [comment]
It is just not anything on a current roadmap or a long term roadmap, so keeping the issue open would give people an impression that it is, and [there are plenty of other issues that have a lot more community support](https://github.com/denoland/deno/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc).

**Comment by hayd** [comment]
with `deno lsp` this may be somewhat more straightforward... I think someone in the community needs to give wiring this up a shot (as kitson points out it's very unlikely to be someone in the deno team)

https://github.com/krassowski/jupyterlab-lsp

**Comment by mmastrac** [comment]
For anyone who stumbles across this post in the future, Deno 1.37 now supports a Jupyter kernel.
